modules/ingress-operator.adoc

```diff
@@ -44,3 +44,17 @@ creates the deployment for the router:
 ----
 $ oc get deployment -n openshift-ingress
 ----
+
+The Ingress Operator uses the `clusterNetwork[].cidr` from the cluster or
+network status to determine what mode (IPv4, IPv6, or dual stack) the managed
+Ingress controller, or router, should operate in. For example, if
+`clusterNetwork` contains only a v6 `cidr`, then the Ingress controller will
```

Comment:
"Ingress controller, or router, should operate in" > "ingress controller, i.e. router, should operate in"